aregion.cpp

```diff
@@ -103,11 +103,13 @@ TownInfo::~TownInfo()
 	if (name) delete name;
 }
 
-void TownInfo::Readin(Ainfile *f, ATL_VER &v)
+void TownInfo::Readin(istream &f)
```

Comment:
As I understand `ATL_VER &v` was not used at all...

Reply:
Correct.  It had been passed down to some but not all functions, and when I chased through, I found that it was not *actually* used anywhere in the reading functions *except* in the very top level to verify that the version was compatable, so I just got rid of it all the way up the call chain(s) from anything below that top level.